src/api/services/usersService.ts

```diff
@@ -67,7 +67,9 @@ export const enrichWithUserFromParams = async (
   if (!user) {
     return res.status(404).send([{ message: 'The user cannot be found.' }]);
   }
-  if (!(await isUserBelongsToParticipant(req.auth?.payload?.email as string, user.participantId))) {
+  if (
+    !(await isUserBelongsToParticipant(req.auth?.payload?.email as string, user.participantId!))
```

Comment:
Just wondering if someone trying to hack and send request to a deleteUserId, will this crash the backend?
For example sending `api/users/:userId/resendInvitation` via postman?

Reply:
It _shouldn't_ be possible to crash the back-end by causing an exception in a route handler - we should have it set up so that all exceptions are intercepted by a top-level handler. If anyone notices a backend crash - please let everyone know and we'll fix it!